### PR TITLE
Get rid of references to Build and Building

### PR DESCRIPTION
**Pull request by CarolynMabbott**
References issue #517 

# Changes

Get rid of references to Build and Building in the code which are no longer used

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/dashboard/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: packages/components/src/components/Rerun/Rerun.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ export class Rebuild extends Component {
           namespace,
           pipelineRunName: newPipelineRunName
         });
-        this.props.setShowRebuildNotification({
+        this.props.setShowRerunNotification({
           message: 'Rebuilt PipelineRun successfully',
```
Comment (a-roberts) [inline packages/components/src/components/Rerun/Rerun.js]: Rerun

File: pkg/endpoints/pipeline.go [hunk 2]
```diff
@@ -206,40 +206,40 @@ func (r Resource) GetPipelineRunLog(request *restful.Request, response *restful.
 	response.WriteEntity(pipelineRunLogs)
 }
 
-func (r Resource) rebuildRun(name, namespace string) (*v1alpha1.PipelineRun, error) {
+func (r Resource) RerunRun(name, namespace string) (*v1alpha1.PipelineRun, error) {
```
Comment (a-roberts) [inline pkg/endpoints/pipeline.go]: Just rerun would be better

File: pkg/endpoints/pipeline.go [hunk 3]
```diff
@@ -88,8 +88,8 @@ type PipelineRunUpdateBody struct {
 	STATUS string `json:"status"`
 }
 
-// PipelineRunRebuild - a name only for now
-type RebuildRequest struct {
+// PipelineRunRerun - a name only for now
```
Comment (a-roberts) [inline pkg/endpoints/pipeline.go]: We can remove the comment as well

File: packages/utils/src/utils/index.js [hunk 4]
```diff
@@ -116,9 +116,7 @@ export function getStatus(resource) {
 }
 
 export function isRunning(reason, status) {
-  return (
-    status === 'Unknown' && (reason === 'Running' || reason === 'Building')
-  );
+  return status === 'Unknown' && reason === 'Running';
```
Comment (a-roberts) [inline packages/utils/src/utils/index.js]: Definitely safe to do? What was `Building` ever used for/set? I don't see a reference to it being removed here

Comment (CarolynMabbott) [reply packages/utils/src/utils/index.js]: Building was what used before it changed to Run/Running
All instances of Building being used were removed before this PR

**Pull request merged**